EPRN.Portal/Areas/Reprocessor/Controllers/PRNSController.cs

```diff
@@ -236,6 +235,33 @@ public async Task<IActionResult> DecemberWaste(DecemberWasteViewModel decemberWa
                 });
         }
 
+        [HttpGet]
+        [ActionName(Routes.Areas.Actions.PRNS.DraftConfirmation)]
+        public async Task<IActionResult> DraftConfirmation(int? id)
+        {
+            if (id == null)
+                return NotFound();
+
+            var viewModel = await _prnService.GetDraftConfirmationViewModel(id.Value);
```

Comment:
Again, what happens if you land on this page with a PRN that is already a draft?

Reply:
Done.